### PR TITLE
Issue 298

### PR DESCRIPTION
**Pull request by jeffsf**
Addresses primary usability concern about localhost outlined in #298 
- Clarified in Accessing the Console that Karaf console only binds to localhost
- Moved -p 8101 to more conventional location in ssh command
- Updated console splash screen to 2.0.0 Release version

Formatting checked in MacDown, but not in context of site itself.

Does not address the "suggested additions" outlined in #298

### CODE REVIEW

File: administration/console.md [hunk 1]
```diff
@@ -16,23 +16,29 @@ The console offers the option to:
 ## Accessing the Console
 
 Accessing the console depends on the start mode of openHAB.
+
+### Started with Script
 The manually start using shell/batch script ends directly in the console.
 
-If openHAB runs a service, the console can be accessed using ssh to the openHAB host on port 8101.
+### Running as Service
+If openHAB runs a service, the console can be accessed using ssh from the openHAB host on localhost:8101. It is intentionally not available from remote hosts due to security concerns.
```
Comment (hakan42) [inline administration/console.md]: Minor nitpick, could you please use one line per sentence? This helps avoiding git history churn 😄

File: administration/console.md [hunk 2]
```diff
@@ -16,23 +16,29 @@ The console offers the option to:
 ## Accessing the Console
 
 Accessing the console depends on the start mode of openHAB.
+
+### Started with Script
 The manually start using shell/batch script ends directly in the console.
```
Comment (ThomDietrich) [inline administration/console.md]: You didn't add this sentence but could you still improve it? As an outsider, I would not understand the meaning...

File: administration/console.md (hunk 1 above)
Comment (ThomDietrich) [inline administration/console.md]: Thanks @hakan42 ;)

You've added localhost:8101, I fear not everyone will understand that notation :-/ Do you agree "locahost, TCP port 8101" would be better?

**Comment by jeffsf** [comment]
Yes on both -- I'll re-read and clarify some of the puzzling wording, as well as try to localize diffs.

**Comment by ThomDietrich** [comment]
Hey @jeffsf, that was quick! :tada: 

Looks great so far ;) Regarding the "suggestions": Username/Password is already there but badly headlined. Check the last line of the article. Regarding the certificate: The solution is available somewhere, I have seen it only a couple of weeks ago. Either here or in the community forum. I have to leave now but I trust you will find it quickly ;)

**Comment by jeffsf** [comment]
The more I read the Console page, the more I believe it could benefit from major revisions and clarifications. I'm going to close this PR and work on that task as I can.

**Comment by ThomDietrich** [comment]
You don't have to close it. It's good to still be aware.
If you want to add more changes in the next couple of days, we can just leave it open and you can write a comment as soon as you are finished.

> The more I read the Console page, the more I believe it could benefit from major revisions and clarifications. 

That's the spirit! 😉

**Comment by jeffsf** [comment]
Will do -- I typically don't rewrite public branches...

Is there a documentation style guide out there for openHAB?

**Comment by ThomDietrich** [comment]
No there is not, even though I wanted to create one at some point. For now please just follow the style of existing articles. Most are in one style I tried to establish. Check e.g. https://github.com/openhab/openhab-docs/blob/gh-pages/installation/linux.md

> I typically don't rewrite public branches...

What do you mean?

**Comment by jeffsf** [comment]
While with gerrit I can replace one set of patches with another, I'm not aware of how to change the _source_ branch of a github pull request.

At least they "fixed" changing the _target_ branch after all these years -- https://github.com/blog/2224-change-the-base-branch-of-a-pull-request

**Comment by ThomDietrich** [comment]
I can't speak for gerrit but on GitHub, you simply add commits to the branch ;)
Check the other PRs for examples

**Comment by ThomDietrich** [comment]
Hey @jeffsf, any news?

**Comment by jeffsf** [comment]
Given that the console provides no value to the average user (logs are available on the filesystem and the console does not provide complete information about OpenHAB entities created with Paper UI) and that I'm still fighting with a deep lack of documentation of the file-based format for system definition, I won't be working further on this.

**Comment by ThomDietrich** [comment]
Too bad. I hope to see you here again soon! You can help out with PRs but also by starting a discussion in the issues section ;)

I have to apologize for not informing you, another user continued where you left off in the meantime, see the PR linked above. We will have an improved console page soon. Feel free to review the intended changes!

**Pull request closed without merging**